build.gradle

```diff
@@ -34,13 +34,14 @@ dependencies {
     implementation("com.github.KittyBot-Org:Lavalink-Client:f7842dfdb4") {
         exclude group: 'com.sedmelluq', module: 'lavaplayer'
     }
-    implementation 'com.github.devoxin:lavaplayer:1.3.59'
+    implementation 'com.sedmelluq:lavaplayer:1.3.53'
```

Comment:
we can go to devoxin's fork again cuz it has some NPE fixes...

Reply:
ffs, this is like threading on thin ice

Reply:
yeah it's so stupid seq should add him as collab & then we have one up2date project again
I'm wirting a message in lavaplayer channel now 
already talked with devoxin about it & we is open for it...